### PR TITLE
Zowe v2 conformance tables update from v1 to v2

### PR DESCRIPTION
**Pull request by jp669844**
Signed-off-by: jp669844 <jan.prihoda@broadcom.com>

### CODE REVIEW

**Comment by jmertic** [comment]
Hi @jp669844 - can you also change the references in the participation form from v1 to v2? For now I did it in my local copy - attached are the built assets. Docusign powerform is at https://na3.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=f6393a5f-c5f4-4ba7-8ebc-3d075b74227c&env=na3&acct=f30e10ec-fea1-4dd8-a262-384a61edabb5&v=2

[Zowe.Conformance.Program.-.Brand.Guidelines.pdf](https://github.com/openmainframeproject/foundation/files/8489752/Zowe.Conformance.Program.-.Brand.Guidelines.pdf)
[Zowe.Conformance.Program.-.Brand.Guidelines.pdf](https://github.com/openmainframeproject/foundation/files/8489752/Zowe.Conformance.Program.-.Brand.Guidelines.pdf)
[Zowe.Conformance.Program.-.Participation.Form.pdf](https://github.com/openmainframeproject/foundation/files/8489754/Zowe.Conformance.Program.-.Participation.Form.pdf)
[Zowe.Conformance.Program.-.Participation.Form.pdf](https://github.com/openmainframeproject/foundation/files/8489754/Zowe.Conformance.Program.-.Participation.Form.pdf)
[Zowe.Conformance.Program.-.Terms.and.Conditions.pdf](https://github.com/openmainframeproject/foundation/files/8489755/Zowe.Conformance.Program.-.Terms.and.Conditions.pdf)
[Zowe.Conformance.Program.-.Terms.and.Conditions.pdf](https://github.com/openmainframeproject/foundation/files/8489755/Zowe.Conformance.Program.-.Terms.and.Conditions.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489756/Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489756/Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489757/Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489757/Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489758/Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489758/Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489759/Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489759/Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf)

**Comment by jmertic** [comment]
Hi @jp669844 - can you also change the references in the participation form from v1 to v2? For now I did it in my local copy - attached are the built assets. Docusign powerform is at https://na3.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=f6393a5f-c5f4-4ba7-8ebc-3d075b74227c&env=na3&acct=f30e10ec-fea1-4dd8-a262-384a61edabb5&v=2

[Zowe.Conformance.Program.-.Brand.Guidelines.pdf](https://github.com/openmainframeproject/foundation/files/8489752/Zowe.Conformance.Program.-.Brand.Guidelines.pdf)
[Zowe.Conformance.Program.-.Brand.Guidelines.pdf](https://github.com/openmainframeproject/foundation/files/8489752/Zowe.Conformance.Program.-.Brand.Guidelines.pdf)
[Zowe.Conformance.Program.-.Participation.Form.pdf](https://github.com/openmainframeproject/foundation/files/8489754/Zowe.Conformance.Program.-.Participation.Form.pdf)
[Zowe.Conformance.Program.-.Participation.Form.pdf](https://github.com/openmainframeproject/foundation/files/8489754/Zowe.Conformance.Program.-.Participation.Form.pdf)
[Zowe.Conformance.Program.-.Terms.and.Conditions.pdf](https://github.com/openmainframeproject/foundation/files/8489755/Zowe.Conformance.Program.-.Terms.and.Conditions.pdf)
[Zowe.Conformance.Program.-.Terms.and.Conditions.pdf](https://github.com/openmainframeproject/foundation/files/8489755/Zowe.Conformance.Program.-.Terms.and.Conditions.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489756/Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489756/Zowe.Conformance.Program.-.Test.Evaluation.Guide.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489757/Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf)
[Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489757/Zowe.Conformance.Program.-.Test.Evaluation.Guide.Table.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489758/Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf](https://github.com/openmainframeproject/foundation/files/8489758/Zowe.Support.Provider.-.Test.Evaluation.Guide.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489759/Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf)
[Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf](https://github.com/openmainframeproject/foundation/files/8489759/Zowe.Support.Provider.-.Test.Evaluation.Guide.Table.pdf)

**Comment by jp669844** [comment]
@jmertic Done, please review.

File: zowe_conformance/participation_form.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Zowe Conformance Program
+# Zowe V2 Conformance Program
```
Comment (jmertic) [inline zowe_conformance/participation_form.md:1]: Let's not edit these - the program regardless of version is the Zowe Confomrance Program. The specific versions are the only things that should change.

File: zowe_conformance/support_provider_evaluation_guide_table.md [hunk 2]
```diff
@@ -1,115 +1,107 @@
-# Zowe Support Provider Conformance Guide
+# Zowe Support Provider v2 Conformance Guide
```
Comment (jmertic) [inline zowe_conformance/support_provider_evaluation_guide_table.md:1]: Same here - the program name is the same, just the version is different

File: zowe_conformance/test_evaluation_guide_table.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
-# Zowe Conformance Test Evaluation Guide
+# Zowe v2 Conformance Test Evaluation Guide
```
Comment (jmertic) [inline zowe_conformance/test_evaluation_guide_table.md:1]: Same comment as in the other docs.

File: zowe_conformance/participation_form.md (hunk 1 above)
Comment (jp669844) [reply zowe_conformance/participation_form.md]: Ok, removed back

File: zowe_conformance/test_evaluation_guide_table.md (hunk 3 above)
Comment (jp669844) [reply zowe_conformance/test_evaluation_guide_table.md]: Resolved

File: zowe_conformance/support_provider_evaluation_guide_table.md (hunk 2 above)
Comment (jp669844) [reply zowe_conformance/support_provider_evaluation_guide_table.md]: Resolved

File: zowe_conformance/support_provider_evaluation_guide_table.md [hunk 4]
```diff
@@ -123,94 +115,95 @@ This guide describes the requirements of the support conformance program. All Ap
...
  <tr>
-   <th style="background-color:#555555">8</th>
-   <th style="background-color:#555555">V1</th>
-   <th style="background-color:#AAAAAA">x</th>
-   <th style="background-color:#AAAAAA"></th>
-   <th></th>
-   <td> <b>Support Provider confirms:</b> Capable Support as defined in item (1)
-   </td>
+   <td style="background-color:#555555">8</td>
+   <td style="background-color:#555555">v3</td>
```
Comment (jmertic) [inline zowe_conformance/support_provider_evaluation_guide_table.md:178]: Should this be v2?

Comment (jp669844) [reply zowe_conformance/support_provider_evaluation_guide_table.md]: Of course, it should, typo.

**Comment by jmertic** [comment]
@jp669844 @RASakach was going to merge in these items, but saw there are some outstanding changes needing handled. Can you adjust?

**Comment by jp669844** [comment]
Per agreement with @RASakach I close this PR and will open a new one.

**Pull request closed without merging**